### ISSUE TITLE
When we have some selections, and tab outsize of the table, the selection should be confirmed

### ISSUE THREAD

**Issue by LiKang6688** [qlik-oss/sn-table#307]

**Comment by haxxmaxx** [comment]
There are already an issue on this, this should most likely be closed
